src/main/resources/static/css/style.css

```diff
@@ -3461,13 +3463,13 @@ table.projects tr td:first-child {
     28. Custom style for stack logs
 -------------------------------------------------------------------*/
 
-code[class*="language-"],
-pre[class*="language-"] {
+code.language-shellOutput,
+pre.language-shellOutput {

```

Comment:
I don't know how, but this rule breaked the stack log view, as the background should be black : 
![image](https://user-images.githubusercontent.com/7531844/62245209-62510080-b3e1-11e9-8b9b-01a1733f0a2a.png)